include/zephyr/bluetooth/hci_types.h

```diff
@@ -296,6 +296,9 @@ struct bt_hci_cmd_hdr {
 #define BT_HCI_GENERAL_BONDING                  0x04
 #define BT_HCI_GENERAL_BONDING_MITM             0x05
 
+/* Clear Bonding flag */
+#define BT_HCI_SET_NO_BONDING(auth) ((auth) & 0x01)
```

Comment:
It doesn't seem to me like this needs to be in a public header file. I.e. I'd move it internal to `ssp.c` or just open code it in the single place you're using it (it really doesn't hurt readability IMO).